provisioner/plain-v0/main.go

```diff
@@ -57,19 +59,26 @@ func main() {
 	var probeAddr string
 	var systemNamespace string
 	var unpackImage string
+	var rukpakVersion bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&systemNamespace, "system-namespace", "rukpak-system", "Configures the namespace that gets used to deploy system resources.")
 	flag.StringVar(&unpackImage, "unpack-image", "quay.io/operator-framework/plain-v0-provisioner:latest", "Configures the container image that gets used to unpack Bundle contents.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&rukpakVersion, "version", false, "Displays rukpak version and git commit")
 	opts := zap.Options{
 		Development: true,
 	}
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	if rukpakVersion {
+		fmt.Print(version.String())
+		os.Exit(0)
+	}
```

Comment:
This is fine for now, but I think it's worthwhile also logging the version of the binary at controller startup so we have that information available for debugging outside of when users supply the --version flag to this binary.

Reply:
And that realistically would just look like the following:

```go
setupLog.Info("starting up the provisioner", zap.String("version", version))
```

Reply:
Great idea, thanks